src/commands/kv/namespace/create.rs

```diff
@@ -21,7 +27,22 @@ pub fn create(target: &Target, user: GlobalUser, title: &str) -> Result<(), fail
     });
 
     match response {
-        Ok(success) => message::success(&format!("Success: {:#?}", success.result)),
+        Ok(success) => {
+            message::success(&format!("Success: {:#?}", success.result));
+            match env {
+                Some(env) => message::success(&format!(
+                    "Add the following to your TOML under [env.{}]:",
+                    env
+                )),
+                None => message::success(&format!("Add the following to your TOML:")),
+            };
+            println!(
```

Comment:
should add a check here to see if `kv-namespaces` already exists, if not, keep current logic, else print "please add this to your kv-namespaces array"